### ISSUE TITLE
openat2  missing syscall issue

### ISSUE THREAD

**Issue by djvs** [falcosecurity/falco#2037]
**Describe the bug**

```Thu Jun  2 18:20:01 2022: Falco version 0.30.0 (driver version 3aa7a83bf7b9e6229a3824e3fd1f4452d1e95cb4)
Thu Jun  2 18:20:01 2022: Falco initialized with configuration file /etc/falco/falco.yaml
Thu Jun  2 18:20:01 2022: Loading rules from file /etc/falco/falco_rules.yaml:
Thu Jun  2 18:20:01 2022: Runtime error: Could not load rules file /etc/falco/falco_rules.yaml: Error loading rules: /usr/share/falco/lua/sinsp_rule_utils.lua:107: Unknown event/syscall "openat2" in filter: open_write and (fd.filename in (shell_config_filenames) or
 fd.name in (shell_config_files) or
 fd.directory in (shell_config_directories))
and not proc.name in (shell_binaries) and not exe_running_docker_save
. Exiting.
```

openat2 syscall (referenced in "open_write" snippet) is not recognized for unknown reasons.

Default falco deployment on kubernetes with what I believe is a more or less standard values file (let me know if you need additional info from this).

**Expected behaviour**

Falco should start.

**Screenshots**

**Environment**

- Falco version:
Falco version 0.30.0 (driver version 3aa7a83bf7b9e6229a3824e3fd1f4452d1e95cb4)

- System info:

- Cloud provider or hardware configuration:
GCP

- OS:
Container-optimized OS

- Kernel:
`Linux [...] 5.4.144+ #1 SMP Sat Sep 25 09:56:01 PDT 2021 x86_64 GNU/Linux`

- Installation method:
Helm chart (`helm install falco falcosecurity/falco -f ./values-falco.yaml -n falco --create-namespace`)

**Additional context**

**Comment by jasondellaluce** [comment]
Hey! I see you are running Falco 0.30, and IIRC openat2 was released in Falco 0.31.0. So this seems to be a case in which you are running a recent ruleset without having updated Falco.

If you're interested in updating Falco, we are releasing 0.32.0 by the end of the week!

**Comment by djvs** [comment]
Ah my mistake, it was hard-coded in my values file.  Thanks!
